app/models/services/pubmed.rb

```diff
@@ -34,10 +36,15 @@ def get_uid_metadata(uid)
   def get_abstract(uid)
     abstract_uri = generate_uri(@abstract_url, @default_parameters.merge(id: uid, retmode: 'xml'))
 
-    abstract = Hash.from_xml(Net::HTTP.get(abstract_uri))['PubmedArticleSet']['PubmedArticle']['MedlineCitation']['Article']['Abstract']['AbstractText']
+    begin
+      abstract = Hash.from_xml(Net::HTTP.get(abstract_uri))['PubmedArticleSet']['PubmedArticle']['MedlineCitation']['Article']['Abstract']['AbstractText']
+    rescue
+      pubmed_scrape = Hash.from_xml(Net::HTTP.get(URI.parse(@pubmed_url + uid + @pubmed_xml_options)))
+      abstract = pubmed_scrape['pre'][/#{"AbstractText"}(.*?)#{"</AbstractText>"}/m, 1].partition('>').last
```

Comment:
Also, you seem to have fragile code in the rescue block, as well. If that code breaks, what happens then?